.github/copilot-instructions.md

```diff
@@ -0,0 +1,89 @@
+# GitHub Copilot Instructions for Literate Programming Project
+
+## Project Overview
+This is a literate programming research project that creates academic documentation about literate programming methodologies. The project uses LaTeX for document preparation and Noweb for literate programming demonstrations.
+
+## Technology Stack
+- **LaTeX**: Primary document preparation system for academic papers and books
+- **Noweb**: Literate programming tool for creating documented code examples
+- **Make**: Build system for compiling LaTeX documents to PDF
```

Comment:
And for extracting compilable source files (e.g. .py) from the literate source files (.nw).